code/__HELPERS/game.dm

```diff
@@ -557,6 +557,11 @@ block( \
 		else
 			return
 
+proc/AddHud(mob/living/carbon/human/character, obj/structure/overmap/ship/simulated/ship)
+	var/faction_hud_type = FACTION_HUD_GENERAL
+	var/faction_hud_name = ship.prefix
+	add_faction_hud(faction_hud_type, faction_hud_name, character)
```

Comment:
is there a reason you made these into variables instead of just: 
```suggestion
	add_faction_hud(ship.prefix, FACTION_HUD_GENERAL, character)
```

Reply:
Originally there was a reason but it was changed down the line and i forgot to incorporate that into it. Originally it let you only see your faction which is why the faction_hud_type was needed no idea on the name though